packages/clarity-js/src/layout/encode.ts

```diff
@@ -69,7 +71,7 @@ export default async function (type: Event, ts: number = null): Promise<void> {
                 }
                 tokens = tokenize(tokens, metadata);
             }
-
+            if (type == Event.Mutation) { timeline.track(eventTime, type); }
```

Comment:
We likely don't need to track this as an entry in Timeline. Can we instead track it inside baseline event? We can discuss this more offline - easier than on PR feedback.

Reply:
And if we don't log it in timeline, timeline can probably stay where it is within Interactions with non-null x & y. It will be specific to interactions.